.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Run Tests
         uses: nick-fields/retry@v2.8.3
         with:
-          command: pnpm run test:ci --base=${{ github.event.before }}
+          command: pnpm run test:ci --base=${{ github.event.before }} --skip-nx-cache
```

Comment:
I remember discussing this for the actual build, which we do:

https://github.com/TanStack/query/blob/eff7f03c44c91d80be1375e9686cf41f9d48a17d/scripts/publish.ts#L360

but this is just running tests. I think we don't need to skip cache here